packages/manager/modules/pci/src/components/hds/template.html

```diff
@@ -23,20 +23,4 @@
             </a>
         </oui-checkbox-description>
     </oui-checkbox>
-
-    <oui-message data-ng-if="!$ctrl.hds.isValidSupportLevel" data-type="info">
-        <span data-translate="pci_projects_hds_banner_info"></span>
```

Comment:
**suggestion:** Both translations `pci_projects_hds_banner_info` and `pci_projects_hds_banner_info_link` can be removed then.

see: packages/manager/modules/pci/src/components/hds/translations/Messages_fr_FR.json

Reply:
If i remember we have a tools which remove no used translation keys.
isn't the case ?

Reply:
You're right! But it only work with translation files. Here we are updating an HTML template.

Reply:
Thanks, removed.